src/async.js

```diff
@@ -7,7 +7,7 @@ function _noop() { /* empty */ }
 /**
  * Iterates an array in series.
  *
- * @param {*[]} array - Array to iterate.
+ * @param {any[]} array - Array to iterate.
```

Comment:
I don't think `any` is a valid jsdoc type, this may need to be `Array.<*>` or something similar.